src/index.js

```diff
@@ -83,6 +94,17 @@ class LndEngine {
       this[configKey] = this.currencyConfig[configKey]
     })
 
+    // 90 minutes, from BOLT #2
+    // see https://github.com/lightningnetwork/lightning-rfc/blob/master/
+    //     02-peer-protocol.md#cltv_expiry_delta-selection
+    this.outboundTimeLock = outboundTimeLock || 9 * this.secondsPerBlock
```

Comment:
should this be 9 blocks, or 90 minutes? (LTC will have a different block time)

Reply:
The formula in the link in the comment calculates the value in terms of blocks so was following that.

Reply:
The link in the comment is for Bitcoin only

Reply:
Yeah, so do you think there is a better approach? The calculation there seems like it could make sense for other blockchains, though the "optimal" value would probably depend on other factors like network difficulty, requiring a custom value for each blockchain.